### PR TITLE
[#37] Add Project Constraints

### PR DESCRIPTION
**Pull request by MannyLemos**
Got rid of duplicate data

### CODE REVIEW

**Review by aaronhsli — APPROVED** [review]
Lgtm!

File: docs/SRS/SRS.md [hunk 1]
```diff
@@ -86,16 +117,53 @@ add content
...
+Real time analysis and response is an integral component of the cyclops ride assist system. More precisely, desired results are only of value if they can be delivered on time every time.  
+
 ### 3.2 Naming Conventions and Definitions
-add content
-### 3.3 Relevant Facts and Assumptions
-add content
-## 4 Context Diagrams
+#### 3.2.1 Naming Conventions
+#### 3.2.2 Constants
+- Gravity = 9.81
```
Comment (amosyu2000) [inline docs/SRS/SRS.md:144]: Include the unit

File: docs/SRS/SRS.md [hunk 2]
```diff
@@ -86,16 +117,53 @@ add content
...
+
+**Assumption 2:**  
+While on the road, cyclists will abide by traffic laws. This means travelling in marked bike lanes where available.  
+**Rationale 2:**  
+The system will not be able to distinguish between parked vehicles, which may appear momentarily in a cyclists blind-spot, and moving vehicles. As a result, if a cyclist is not travelling in designated bike lanes they may be subject to increased instances of false blind-spot detection triggers.  
+
+## 4 Context Diagrams  
+![System Context Diagram](https://user-images.githubusercontent.com/68861121/193955661-d965823f-079b-444f-9cb3-cf34a120ac87.png)
 ## 5 Functional Decomposition Diagrams
+![Functional Decomposition Diagram](https://user-images.githubusercontent.com/68861121/193956013-84d3a52b-fdcb-43fb-8c86-9ae85c845518.png)
```
Comment (amosyu2000) [inline docs/SRS/SRS.md:166]: I made this diagram and it is wrong. Use this one instead 
```
![image](https://user-images.githubusercontent.com/46848538/194113559-4d5de21a-f88a-4fbd-b3b5-558473bf8b25.png)
```

image

**Review by amosyu2000 — DISMISSED** [review]

File: docs/SRS/SRS.md (hunk 2 above)
Comment (amosyu2000) [reply docs/SRS/SRS.md:166]: image

```
![image](https://user-images.githubusercontent.com/46848538/194126664-519ac85c-9113-4155-bf91-f70dd4599b53.png)
```

Comment (amosyu2000) [reply docs/SRS/SRS.md:166]: image
```
![image](https://user-images.githubusercontent.com/46848538/194127274-453bbdc3-935e-4b3d-8f8f-333d7f1dcf6e.png)
```

Comment (amosyu2000) [reply docs/SRS/SRS.md:166]: image
```
![image](https://user-images.githubusercontent.com/46848538/194128815-19b91083-7090-4817-b8f0-cf82e7fdef75.png)
```

Comment (amosyu2000) [reply docs/SRS/SRS.md:166]: Actually no, I'll add it myself during my own PR.

**Pull request merged**